search/search_integration_tests/processor_test.cpp

```diff
@@ -2207,5 +2207,73 @@ UNIT_CLASS_TEST(ProcessorTest, SynonymMisprintsTest)
     TEST(ResultsMatch("se street", rules), ());
   }
 }
+
+UNIT_CLASS_TEST(ProcessorTest, TestCityPostcodes)
+{
+  string const countryName = "France";
+
+  TestVillage marckolsheim(m2::PointD(0, 0), "Marckolsheim", "en", 10 /* rank */);
+  marckolsheim.SetPostcode("67390");
+
+  TestStreet street(
+      vector<m2::PointD>{m2::PointD(-0.5, 0.0), m2::PointD(0, 0), m2::PointD(0.5, 0.0)},
+      "Rue des Serpents", "en");
+
+  TestBuilding building4(m2::PointD(0.0, 0.00001), "", "4", street.GetName("en"), "en");
+
+  auto countryId = BuildCountry(countryName, [&](TestMwmBuilder & builder) {
+    builder.Add(marckolsheim);
+    builder.Add(street);
+    builder.Add(building4);
+  });
+
+  SetViewport(m2::RectD(-1, -1, 1, 1));
+  {
+    Rules rules{ExactMatch(countryId, building4), ExactMatch(countryId, street)};
+    // Test we not require building to have postcode if city has.
```

Comment:
s/Test we/Test that we/
s/not/do not/
the building, the city

Reply:
done